source/_integrations/unifiprotect.markdown

```diff
@@ -274,6 +274,32 @@ Two URLs for proxy API endpoints:
 
 The easiest way to find the `nvr_id`, `camera_id`, `start`, and `end` times is by viewing one of the videos from UniFi Protect in the Media browser. If you open the video in a new browser tab, you will see all these values in the URL. The `start` time is the last_changed timestamp of the event when the sensor started detecting motion. The `end` time is the last_changed timestamp of the event when the sensor stopped detecting motion. Similarly, to see the `event_id` of the image, go to {% my developer_states title="**Developer Tools** > **States**" %} and find the event when the sensor started detecting motion.
 
+### Example Notification Automation with Video
+
+```yaml
+alias: "Security: Camera Motion Notification"
+description: "Sends a notification with video upon motion detection."
+triggers:
+  - entity_id:
+      - binary_sensor.g5_bullet_motion # Replace with your camera entity
+    trigger: state
+    from: "on"
+    to: "off"
+actions:
+  - data:
+      message: "Motion detected at Camera XXX"
+      data:
+        image: >-
+          {% raw %}/api/unifiprotect/thumbnail/{{ config_entry_id(trigger.entity_id) }}/{{ trigger.from_state.attributes.event_id }}{% endraw %}
+        video: >-
+          {% raw %}/api/unifiprotect/video/{{ config_entry_id(trigger.entity_id) }}/{{ trigger.from_state.attributes.event_id }}{% endraw %}
+    action: notify.mobile_app_your_device # Replace with your notification target
+mode: single
+max_exceeded: silent
+```
+
+It's important to wait for the motion sensor to change from `on` to `off` before sending the notification. This ensures that the event has ended and the video is accessible; otherwise, you may get an error instead of the video link.
```

Comment:
```suggestion
Waiting for the motion sensor to change from `on` to `off` before sending the notification is essential. Waiting ensures that the event has ended and the video is accessible; otherwise, you may get an error instead of the video link.
```